docs/src/image_loading.md

```diff
@@ -0,0 +1,203 @@
+# Image Loading
+
+## Overview
+
+The Image Loading module is a component of the MCU Runtime SDK designed for managing SOC images stored in flash memory. This module provides APIs for:
+
+- Detecting SOC images availability in the [flash storage](./image_loading.md).
+- Parsing the [SOC manifest](https://github.com/chipsalliance/caliptra-sw/blob/main/auth-manifest/README.md).
+- Loading SOC images to a target component.
+
+The diagram below illustrates an **example** how the Caliptra subsystem, integrated within the SOC, is used to load SOC images to vendor components. In this example, these are the 2 Vendor CPUs and their corresponding RAMs.
+
+<p align="center">
+    <img src="images/image_loading_sample.svg" alt="flash_config" width="50%">
+</p>
+
+Pre-image loading state:
+
+* Caliptra MCU Runtime Firmware (MCU RT) is running in the MCU.
+* The SOC target components (Vendor CPU 1 and 2) are held in reset.
+
+Image Loading Steps:
+
+1. MCU RT checks for available SOC images in the [flash storage](./image_loading.md).
+2. MCU RT authorizes the [SOC manifest](https://github.com/chipsalliance/caliptra-sw/blob/main/auth-manifest/README.md) and the SOC images by following steps described in [Manifest-Based Authorization](https://github.com/chipsalliance/caliptra-sw/blob/main-2.x/runtime/README.md#manifest-based-image-authorization-new-in-12) section of the Caliptra Runtime Specification.
```

Comment:
Will MCU RT authorize the SOC manifest directly or forward the authentication request to Caliptra core?

Reply:
After retrieving the SOC manifest from flash, the MCU RT application will call another API to authenticate and forward the manifest to the Caliptra core.